src/workerd/api/global-scope.c++

```diff
@@ -727,6 +728,136 @@ jsg::JsString ServiceWorkerGlobalScope::btoa(jsg::Lock& js, jsg::JsString str) {
       strArray.asChars().begin(), strArray.size(), result.asChars().begin());
   return js.str(result.first(written));
 }
+
+//Fuzzilli required macros
```

Comment:
needs to go to a separate fuzzily.h and properly guarded not to be included if fuzilly is not enabled